src/core/server/execution_context/execution_context_service.ts

```diff
@@ -124,7 +124,7 @@ export class ExecutionContextService
     // we have to use enterWith since Hapi lifecycle model is built on event emitters.
     // therefore if we wrapped request handler in asyncLocalStorage.run(), we would lose context in other lifecycles.
     this.contextStore.enterWith(contextContainer);
-    this.log.debug(`set the execution context: ${JSON.stringify(contextContainer)}`);
```

Comment:
Don't think it provides any valuable information. Also, we can consider lowering log level to `trace` to reduce noise in the verbose mode.